frontend/src/bundles/workouts/pages/workout-item/workout-items.tsx

```diff
@@ -29,6 +32,14 @@ const WorkoutItem = (): JSX.Element => {
             <WorkoutTitle workout={currentWorkout} />
             <WorkoutRoute workout={currentWorkout} />
             <WorkoutStats workout={currentWorkout} />
+            {currentWorkout.provider === OAuthProvider.STRAVA && (
+                <Link
+                    to={AppRoute.STRAVA}
+                    className="text-strava-brand mt-[0.5rem] text-right"
+                >
```

Comment:
Do you think this will be enough? Maybe we should point this link to specific workout?
> You must link back to all original Strava data sources presented in your application using the following text format, “View on Strava”.

Reply:
I just added a link to all user's workouts